apps/communications/dialer/elements/call-info-view.html

```diff
@@ -14,6 +14,8 @@ <h1 id="call-info-title"></h1>
 
       <div id="call-durations">
       </div>
+
+      <a id="show-more" data-l10n-id="showMore"></a>
```

Comment:
@DouglasSherk , We will not have to display the "show more" button if there are less than 9 rows and if we apply the above method, I think it's not possible ?